bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/model/SomfyTahomaDeviceDefinition.java

```diff
@@ -25,19 +25,28 @@
 public class SomfyTahomaDeviceDefinition {
 
     private ArrayList<SomfyTahomaDeviceDefinitionCommand> commands = new ArrayList<>();
+    private ArrayList<SomfyTahomaDeviceDefinitionState> states = new ArrayList<>();
 
     public ArrayList<SomfyTahomaDeviceDefinitionCommand> getCommands() {
         return commands;
     }
 
+    public ArrayList<SomfyTahomaDeviceDefinitionState> getStates() {
```

Comment:
```suggestion
    public List<SomfyTahomaDeviceDefinitionState> getStates() {
```

also above. or do you require the list to be an `ArrayList`?

Reply:
OK